docs/plugins/lyrics.rst

```diff
@@ -113,3 +121,19 @@ After that, the lyrics plugin will fall back on other declared data sources.
 
 .. _pip: http://www.pip-installer.org/
 .. _BeautifulSoup: http://www.crummy.com/software/BeautifulSoup/bs4/doc/
+
+Activate on-the-fly translation
```

Comment:
Matching our usual style, this should probably be in Title Case: "Activate On-the-Fly Translation"